app/Data/Web3/Web3NftData.php

```diff
@@ -28,6 +28,7 @@ public function __construct(
         public ?string $collectionImage,
         public ?string $collectionWebsite,
         public ?string $collectionDescription,
+        public ?string $bannerImageUrl,
```

Comment:
would rename it to `collectionBannerImageUrl` so it's in line with the other collection items here to avoid confusion